app/models/user.rb

```diff
@@ -11,10 +9,43 @@ class User < ActiveRecord::Base
          :trackable,
          :validatable
 
+  after_create :send_admin_mail
+
   belongs_to :agency
 
-  scope :case_workers_by_agency, ->(agency = nil) { case_workers.where(agency: agency) }
-  scope :case_workers, -> { where(role: :case_worker) }
+  def self.case_workers_by_agency(agency = nil)
```

Comment:
Rails standards actual prefer using class methods.  Scopes are a thing of the past...when you chain scopes dealing with the same column, like: `Post.published_before(date).published_after(date)` - the 2nd scope will completely override the first one, which is something that does not happen when using class methods.